Sources/ModerationCenter.php

```diff
@@ -1274,9 +1273,8 @@ function list_getWatchedUsers($start, $items_per_page, $sort, $approve_query, $d
 
 		$request = $smcFunc['db_query']('', '
 			SELECT MAX(m.poster_time) AS last_post, MAX(m.id_msg) AS last_post_id, m.id_member
-			FROM {db_prefix}messages AS m' . ($user_info['query_see_board'] == '1=1' ? '' : '
-				INNER JOIN {db_prefix}boards AS b ON (b.id_board = m.id_board AND {query_see_board})') . '
-			WHERE m.id_member IN ({array_int:member_list})' . (!$modSettings['postmod_active'] || allowedTo('approve_posts') ? '' : '
+			FROM {db_prefix}messages AS m
+			WHERE {query_see_message} and m.id_member IN ({array_int:member_list})' . (!$modSettings['postmod_active'] || allowedTo('approve_posts') ? '' : '
```

Comment:
This AND clause should be capitalized and moved to a new line with proper indentation.